README.md

```diff
@@ -1,10 +1,18 @@
+
+<p align="center" width="100%">
```

Comment:
This is what it looks like: 
<img width="1113" alt="image" src="https://github.com/warm-metal/csi-driver-image/assets/34534103/b6f05ff4-b97d-4200-8871-f9bf7976ff8d">

Rendered Markdown: https://github.com/warm-metal/csi-driver-image/blob/029d2f8f7c7bbd83ad6f527b20f3a443675f2126/README.md

Reply:
(updated logo)
This is what it looks like in the README:
![image](https://github.com/warm-metal/container-image-csi-driver/assets/34534103/a468089b-0db3-4b7a-ab3e-6a84cfcf19a0)